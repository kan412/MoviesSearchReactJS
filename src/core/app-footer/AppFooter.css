
.footer{
    width: 100%;
    background:#373737;
    padding:20px 0;
}

.footer a, 
.footer a:link,
.footer a:active,
.footer a:hover{
    text-decoration: none;
    color: unset;
}

.footer .inner{
    max-width: 1200px;
    padding: 0 15px;
    margin: 0 auto;
    text-align: center; 
}

.logo{
    padding: 20px;
    font-size: 25px;
    line-height: 37px;
    color: #f65261;
    font-weight: bold;
}

.logo span{
    font-weight: 400;
}
