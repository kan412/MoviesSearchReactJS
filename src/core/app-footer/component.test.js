import React from 'react';
import { shallow } from 'enzyme';
import AppFooter from './component';

describe('<AppFooter />', () => {
  it('render without breaking', () => {
    const component = shallow(<AppFooter />);
    expect(component).toMatchSnapshot();
  });
});
