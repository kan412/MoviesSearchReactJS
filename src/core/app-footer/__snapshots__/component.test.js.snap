// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppFooter /> render without breaking 1`] = `
<footer
  className="footer"
>
  <div
    className="inner"
  >
    <a
      href="/"
    >
      <span
        className="logo"
      >
        Netflix
        <span>
          roulette
        </span>
      </span>
    </a>
  </div>
</footer>
`;
